src/main/resources/db/migration/V01_005__hierarchy_CL_0000737_Prodcom_Elements.sql

```diff
@@ -0,0 +1,20 @@
+insert into hierarchy (id, name, type) values ('CL_0000737', 'Prodcom Elements', 'classification');
```

Comment:
Does Flyway automatically wrap these files in a transaction? If not, should we? If nothing else, that might make it faster to load. (We could also look into reusing the COPY FROM command if this is a bottleneck).